packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -187,10 +271,48 @@ class CloudFormation {
     } else {
       newEvents = events;
     }
-    showEvents(_.uniqBy(newEvents, 'EventId'));
+    if(this.eventMap) {
+      this.showEventProgress(_.uniqBy(newEvents, 'EventId'));
+    }
+    else {
+      showEvents(_.uniqBy(newEvents, 'EventId'));
+    }
+    
     this.stackEvents = [...allShownEvents, ...newEvents];
   }
 
+  // For each event update the corresponding progress bar
+  showEventProgress(events) {
+    events = events.reverse();
```

Comment:
this is probably close enough that it doesn't really matter, but events are not guaranteed to be in reverse chronological order. They may be out of order as everything is happening async